server/csharp/TicketHub/Authorization/OpaAuthorizationAttribute.cs

```diff
@@ -5,14 +5,65 @@
 
 namespace TicketHub.Authorization;
 
-// Custom authorization attribute, allowing us to template in the fields our
-// policy needs in the input.
-public class OpaAuthorizationAttribute : ActionFilterAttribute
+// Custom authorization attribute, implemented as an authorization filter.
+// This design allows us to template in the fields our policy needs
+// in the input.
+public class OpaRuleAuthorizationAttribute : Attribute, IAsyncAuthorizationFilter
+{
+    private readonly string _rule;
```

Comment:
It's called [`path` below](https://github.com/StyraInc/styra-demo-tickethub/pull/295/files#diff-2555707e83740778fd0d42781b57712f67fa8e2f11ae76075a033d860c73a66aR66), isn't it? Personally, I think I prefer path, but as long as it's consistent I'm OK with it. Or is there some other logic at play here that suggests to use `rule` for this attribute?

Reply:
I'm fine with either choice-- but for devs who are less familiar with OPA, I'd argue that "path" isn't quite as helpful as "rule". Perhaps a mix of the two is better? `rulePath`, maybe?

Reply:
Converged on `_path` for now. Thanks for calling out the naming inconsistency!